templates/spark/v1/spark.yaml

```diff
@@ -0,0 +1,6 @@
+imports:
```

Comment:
Is the idea that this PR represents an example, or that because it's templated, it's instantiating from an example?